src/components/atoms/TextInput/TextInput.tsx

```diff
@@ -36,6 +36,7 @@ export type InputFormValue = InputDomValue | null;
 
 export interface TextInputProps
   extends Omit<ComponentPropsWithoutRef<"input">, "size"> {
+  error?: string;
```

Comment:
if we are using `error` as a boolean, better name it to `hasError`

Reply:
I thought we could use it to show error anywhere if needed.